test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Order.cshtml

```diff
@@ -52,6 +52,9 @@
                 <option value="Credit">Credit</option>
                 <option value="Check">Check</option>
             </select>
+            <select>
+                <option value="Check">Check</option>
```

Comment:
Hu? It verifies it, in the generated output the first is `selected` the second is not.